REVIEW_PROCESS.md

```diff
@@ -0,0 +1,63 @@
+
+There is no rule of thumb on how long the review needs to be. There is also no rule of thumb on how many comments the author must respond to. All that matters is that two reviewers must ultimately approve the article. Reviewers are encouraged to be reasonable and understand that not all of their comments will be heeded. 
+
+Reviewers have **three weeks** to complete their review. After three weeks, it is up to the authors’ discretion whether they wish to give the tardy reviewer more time, or select a new reviewer.
+
+If an article fails to achieve the approval of two reviewers in a reasonable time frame, the PR will be closed. (Who decides this, and how, will be played by ear.) The author may wish to re-submit a modified version of the article. Authors are discouraged from simply re-submitting the same article with more agreeable reviewers. In the event of disagreement on a data standard, the disagreeing parties are encouraged to meet—preferably over food and drink—and have one or more in-person discussions to try and achieve consensus. If consensus is simply impossible, the disagreeing parties should each write an article arguing for their position. The two articles should link to each other on the website, and should respond to each other’s ideas.
+
+---
+
+# Review template
+
+The questions below do not need to be answered line-by-line. Rather, they are meant to convey a general sense of the purpose of that section. It is up to the reviewers to decide how thoroughly they wish to address each section.
+
+## 1. The Big Picture
+
+Is this article appropriate for the Janelia Data Standards project? Is the general premise of the article sound? Is the proposed data standard generalizable beyond that author’s particular use case?
+
+## 2. Technical Choices
+
+Is the proposed standard elegant, straightforward, and focused? Does the author adequately explain the rationale behind the standard? Is the author making any flawed implicit assumptions, either about the problem or about their audience? Does the author provide an implementation and/or example data?
+
+## 3. Writing Style
+
+Is the post readable to bio-imaging developers who may come from a different sub-field, or code in a different language? Does it contain any typos or awkward sentences? Are the ideas organized in a logical flow?
+
+---
+
+Here is a diagram illustrating the review process.
+
+![A sketch of the review process.](review_process_image.png)
+
+---
+
+# Modifying existing articles
+
+Ideas and standards evolve over time, so some articles will need to be modified after publication. The modification history will be noted clearly on the post itself, so that no modification will be silent.
+
+If the author is making a *minor* modification to their own article, they may skip the review process. A minor modification does not change the audience or the use case for the standard, and poses no danger of rendering existing implementations of the standard obsolete.
```

Comment:
Is an article a standard? The two terms are used interchangeably in this section.

Reply:
Upon reflection, I think they are not interchangeable, and they are not used interchangeably, but the distinction is subtle. Article means article, but standard means "the conventions and recommendations at the heart of the article". So the article can be modified for typos, but that doesn't change the standard. And for example, the standard can be made more permissive, but it's weird to talk about an article becoming more permissive. Perhaps "convention" would be better than "standard"? I am open to suggestions.

Reply:
So, in your view, a "standard" is a concept and an "article" is *the* written representation of this concept, is this correct?
I understood standard as (an idea in the form of) a document that is separate from articles posted here. In that sense, a standard *is* an article for me and I misunderstood this section. But I agree, the semantics are pretty subtle here.